src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier;
+
+    public Lottery() {
+        this.colorSupplier = new ColorSupplier();
+    }
+
+    public Ball getRandomBall() {
+        final Colors randomColor = colorSupplier.getRandomColor();
+        final Random randomNumber = new Random();
```

Comment:
once again - I didn't mean to use `final` for variable - please use class-level constants.